clio-util/src/main/scala/org/broadinstitute/clio/util/json/ModelAutoDerivation.scala

```diff
@@ -117,6 +118,10 @@ trait ModelAutoDerivation extends AutoDerivation {
 
   implicit val decodeSymbol: Decoder[Symbol] =
     Decoder.decodeString.map(Symbol.apply)
+
+  def dropNullsFromJson(json: Json): Json = {
+    parse(json.pretty(implicitly)).getOrElse(Json.obj())
+  }
```

Comment:
I'd put this in [JsonOps](https://github.com/broadinstitute/clio/blob/develop/clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchUtil.scala#L50) as an extension method, and instead of jumping through the printer I'd use a `filter` [like this](https://github.com/broadinstitute/clio/blob/develop/clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/RecoveryIntegrationSpec.scala#L225), to avoid dependencies on tangential implicits / to (probably) be more efficient.